aioshelly/coap.py

```diff
@@ -43,14 +52,20 @@ def __init__(self, sender_addr, payload: bytes):
             self.payload = None
 
 
-def get_all_ips():
-    """Get all ip from ethernet interfaces."""
-    ip_list = []
-    for iface in netifaces.interfaces():
+def get_network_objects(objtype: str):
+    """Get all ip/iface from system interfaces."""
+    obj_list = []
+    ifaces = netifaces.interfaces()
+    ifaces.remove("lo")
```

Comment:
Is this always there, even on Windows ?

Reply:
I don't have a windows system with python, but as far as I got it, "lo" is an internal name assigned by netifaces to localhost.

Reply:
On windows this function returns a list of GUIDs of the interfaces which does not make sense.
In my opinion you don't need to filter lo anyhow.